core/src/main/scala/org/apache/spark/storage/RDDInfo.scala

```diff
@@ -21,13 +21,14 @@ import org.apache.spark.annotation.DeveloperApi
 import org.apache.spark.rdd.RDD
 import org.apache.spark.util.Utils
 
+trait InMemoryObjectInfo
```

Comment:
Actually I used it in https://github.com/apache/spark/pull/2851/files#diff-dc63d4ab5b2ed5a2135c054be78820eeR47

I need a common base class to save RDD and Broadcast Info ...